### DISCUSSION TITLE
Change the number of child processes

### DISCUSSION THREAD

**Discussion by AxellioNeal** [zeek/zeek#3156] [General] [answered]
When I run zeek from the command-line to process a pcap using -cr, I can see in top that there are 30 processes running.  For system tuning, I'd like to reduce that to 1 thread and then go as high as is supported.

Is there an option for this or a specific place in the code to make those changes that can then be compiled?

**Comment by bbannier** [comment, marked as answer]
We do not really provide support in GitHub discussions; a better place for this would be our [Discourse forum](https://community.zeek.org/).

That being said, I assume you are running Zeek directly, e.g., with `zeek -Cr ...`. The 30 "processes" you are seeing are in fact threads, most of which sit dormant and only wake up to react to asynchronous activity. Processing in Zeek still happens on a single thread.

**Reply by AxellioNeal** [reply]
Thank you!
